### ISSUE TITLE
Support extra volumes for Kubeadm

### ISSUE THREAD

**Issue by vinceho** [kubernetes/minikube#2767]
This is a feature request.

Support for configuring these fields in the kubeadm.yaml configuration file (implemented by https://github.com/kubernetes/kubeadm/issues/441).

```
APIServerExtraVolumes
ControllerManagerExtraVolumes
SchedulerExtraVolumes
```

This will help support extra options like `--extra-config=apiserver.token-auth-file`. Currently, this does not work because the `kube-apiserver` docker container could not find the file. The path of the token file must be mounted first.

**Environment**:

- **Minikube version**: v0.26.1
- **OS**: Ubuntu 16.04
- **VM Driver**: virtualbox
- **ISO version**: v0.26.0

**Comment by oz123** [comment]
This is already implemented in https://github.com/kubernetes/kubernetes/pull/49840

**Comment by tstromberg** [comment]
Is this issue obsolete then?

**Comment by tstromberg** [comment]
Closing this bug as it hasn't seen activity in awhile, and there does not appear to be any plans to address it. Please re-open if interested.

**Comment by mecampbellsoup** [comment]
> This is already implemented in [kubernetes/kubernetes#49840](https://github.com/kubernetes/kubernetes/pull/49840)

Hi guys - @tstromberg @vinceho @oz123 can you please elaborate on how this could be used with minikube? Is it possible to pass it as an additional option to `minikube start` e.g.?
